lib/algos/list/basic.js

```diff
@@ -1,12 +1,14 @@
 'use strict'; // eslint-disable-line strict
 
+const Extension = require('./Extension').default;
+
 const checkLimit = require('./tools').checkLimit;
 const DEFAULT_MAX_KEYS = 10000;
 
 /**
  *  Class of an extension doing the simple listing
  */
-class List {
+class List extends Extension {
```

Comment:
Let's keep it as-is. @rahulreddy let me know if you want to open an issue for this.